examples/terraform/aws/variables.tf

```
@@ -166,3 +166,8 @@ variable "static_workers_count" {
   description = "number of static workers"
   default     = 0
 }
+
+variable "initial_machinedeployment_spotinstances" {
+  description = "use spot instances for initil machine-deployment"
+  default     = true
```

I think that we should set the default value to false and then override the value in E2E tests.

```suggestion
  default     = false
```

---

Why?

---

OK done